tests/test-timber-integration-acf.php

```diff
@@ -106,6 +106,9 @@ public function testACFFieldObject()
 
     public function testACFFormatValue()
     {
+        // Avoid deprecation warning on ACF 5.6.x
+        remove_filter('acf_the_content', 'wp_make_content_images_responsive');
```

Comment:
Shouldn’t this filter removal be temporary to avoid side effects?

Reply:
Absolutely! good catch 😄